client/blocks/plugins-update-manager/index.tsx

```diff
@@ -57,8 +59,8 @@ export const PluginsUpdateManager = ( props: Props ) => {
 			<MainComponent wideLayout>
 				<NavigationHeader
 					navigationItems={ [] }
-					title="Plugin updates manager"
-					subtitle="Effortlessly schedule plugin auto-updates with built-in rollback logic."
+					title={ translate( 'Plugin updates manager' ) }
+					subtitle={ translate( 'Schedule automatic plugin updates' ) }
```

Comment:
Set up automated schedules so plugins update when you want them to.

Reply:
This copy was reviewed and approved by the copywriting team. Why it needs to be changed?

Reply:
I also think that there is a space to be more specific, but I would leave it for the next PRs since I'm going to deploy this one.

Reply:
@bogiii Could I ask you to hold off if we plan to make changes to translation strings? We should avoid sending strings to translation (and pay for those translations) when there's a good chance they'll get changed again

Reply:
Ouch. I've deployed it.
I wanted to avoid resolving future conflicts and deployed it.